packages/design-system/src/components/OcBreadcrumb/OcBreadcrumb.vue

```diff
@@ -132,6 +161,12 @@ export default defineComponent({
         return [...AVAILABLE_SIZES, 'remove'].some((e) => e === value)
       }
     },
+
+    maxWidth: {
+      type: Number,
+      required: false,
+      default: 99999
```

Comment:
Default looks a little off, can't we change that too 100 or if it needs to be a magic number -1 + add a comment ?